db/schema.rb

```diff
@@ -39,40 +35,32 @@
     t.index ["phone_number"], name: "index_users_on_phone_number", unique: true
   end
 
-  create_table "voters", primary_key: "sos_id", id: :serial, force: :cascade do |t|
+  create_table "voters", primary_key: "reach_id", id: :string, force: :cascade do |t|
+    t.string "sos_id"
     t.string "last_name"
     t.string "first_name"
     t.string "middle_name"
+    t.string "suffix"
+    t.string "prefix"
     t.integer "age"
     t.string "gender"
+    t.string "email"
+    t.string "phone_country_code"
     t.string "primary_phone_number"
     t.string "voting_street_address"
+    t.string "voting_state"
     t.string "voting_city"
     t.string "voting_zip"
-    t.string "support_score"
-    t.string "vote_plan"
+    t.string "voter_registration"
```

Comment:
Is this supposed to be party registration, or the field where we indicate if they're PA HD 82 registered or one of the following based on manual input from the RO: Out of state registered, out of district but in state registered, or unregistered

Reply:
if its party registration, maybe change this to party_id and add a new field for voter_registration_type?

Reply:
Gotcha, this was meant to be 'D`/`R` so I'll check it like you suggest